next.clayui.com/content/docs/css/components/cards.md

```diff
@@ -0,0 +1,1955 @@
+---
+title: "Cards"
+description: "Cards are a specific form of data visualization focused mainly on displaying images."
+---
+
+<div class="nav-toc">
+
+- [Example](#example)
+- [Content Types](#content-types)
+    - [Body](#body)
+    - [Captions](#captions)
+    - [Images](#images)
+    - [Header and footer](#header-and-footer)
+    - [Dividers](#dividers)
+- [Variations](#variations)
+    - [Image Card](#image-card)
+    - [File Card](#file-card)
+    - [User Card](#user-card)
+    - [Horizontal Card](#horizontal-card)
+    - [Interactive Card](#interactive-card)
+- [States](#states)
+    - [Hover](#hover)
+    - [Active](#active)
+    - [Empty](#empty)
+- [Helpers](#helpers)
+
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	Check the <a href="https://liferay.design/lexicon">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/cards/">Cards Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
+</div>
+
+<div class="clay-site-alert alert alert-warning">
+	See the implementation of the <a href="/docs/components/cards.html">Cards component in React</a> following the Lexicon <a href="https://liferay.design/lexicon/core-components/cards/">Cards Pattern</a>.
+</div>
+
+# Example
```

Comment:
hey @diegonvs can you take care of sending another PR by changing this to `h2`? all headings should start with `h2`.